tests/console/yast2_lan.pm

```diff
@@ -19,9 +19,10 @@ use testapi;
 use utils;
 use y2lan_restart_common;
 use version_utils ':VERSION';
-use utils 'zypper_call';
+use y2_common 'is_network_manager_default';
 
 sub handle_Networkmanager_controlled {
+    assert_screen "Networkmanager_controlled";
```

Comment:
Hmm, now I've found `accept_warning_network_manager_default` in y2_common, which is little weird though. It's not like we could simply reuse it here =(

Reply:
Problem with the approach you've proposed is that we then mask error. For scenarios where we expect NM to be there we will not fail if there is wicked and wise-versa.
I would suggest having strict expectations, but for defined scope. So, we can apply you suggestion for generic case not to reduce existing coverage, but for tests which are under our control (YaST job group, yast2_ncurses), we can set it strictly, by splitting test into 2: (one for wicked, one for NM) and scheduling correct one when needed.
For the scope of the ticket you currently work on, it's enough to do what you have found out. It will improve yast2_lan test in the sense that if there is NM, we expect pop-up. So +1.